assets/src/edit-story/components/reorderable/reorderableSeparator.js

```diff
@@ -55,9 +55,11 @@ function ReorderableSeparator({ position, ...props }) {
   if (!isReordering) {
     return null;
   }
-
   return (
-    <Wrapper onPointerEnter={handlePointerEnter} {...props}>
+    // Disable reason: This one does not need keyboard interactivity
+    //  - there are better ways to reorder using keyboard.
+    // eslint-disable-next-line jsx-a11y/mouse-events-have-key-events
+    <Wrapper onMouseOver={handlePointerEnter} {...props}>
```

Comment:
Had to switch this one to `mouseover`. The problem was, that with a scrolling container, a new separator might scroll in under the cursor, but without the cursor actually leaving and entering the separator, it would not correctly register as the current one.

With mouse over, this is fixed, as this event also triggers, when the element moves under the mouse, and not just when the mouse moves over the element.

This might have issues for touch devices, but we'll have to revisit then.